.changeset/swift-pumpkins-knock.md

```diff
@@ -0,0 +1,6 @@
+---
+'@verdaccio/auth': patch
+'verdaccio-htpasswd': patch
```

Comment:
@juanpicado let me know if `verdaccio-htpasswd` should be minor due to the new configuration value. Since the final Verdaccio@6 has not been released I kept it as `patch`.

Reply:
it's fine as minor, patch, the alpha number will increase anyway, the unique difference happens on the package changelog so far.